packages/components/Stack/src/Stack.tsx

```diff
@@ -28,11 +28,10 @@ const render = (Slots: ISlots<IStackSlotProps>, renderData: IStackRenderData, ..
     children = React.Children.map(children, (child: React.ReactChild, index: number) => {
       if (React.isValidElement(child) && index > 0) {
         const childProps = child.props;
-        const extraProps = { style: _mixinStyle(childProps[_styleKey], extraStyle) };
-        return React.cloneElement(child, {
-          ...childProps,
-          ...extraProps,
-        });
+        const mixedStyle = _mixinStyle(childProps[_styleKey], extraStyle);
+        const styleProp = Object.create({});
+        Object.defineProperty(styleProp, 'style', { value: mixedStyle, configurable: true });
```

Comment:
Nit: Maybe add a comment about why these two lines are needed with an issue # ?

Reply:
Solution has changed, since this actually didn't fix the bug.